repository.go

```diff
@@ -62,6 +63,7 @@ var (
 	ErrUnableToResolveCommit     = errors.New("unable to resolve commit")
 	ErrPackedObjectsNotSupported = errors.New("packed objects not supported")
 	ErrSHA256NotSupported        = errors.New("go-git was not compiled with SHA256 support")
+	ErrAlternatePath             = errors.New("alternate path cannot start with http, https or ssh")
```

Comment:
In theory, users could ["install"](https://github.com/go-git/go-git/blob/master/plumbing/transport/client/client.go#L25) other protocol/schemes. So it may be more accurate to call out what is supported instead:

```suggestion
	ErrAlternatePathNotSupported             = errors.New("alternate path must use the file scheme")
```

Reply:
make sense